manage-operational-layers/src/main/java/com/esri/arcgismaps/sample/manageoperationallayers/screens/MainScreen.kt

```diff
@@ -23,28 +23,30 @@ import androidx.compose.foundation.layout.padding
 import androidx.compose.material3.Scaffold
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
-import com.esri.arcgismaps.sample.manageoperationallayers.components.ComposeMapView
+import androidx.compose.ui.platform.LocalContext
+import com.arcgismaps.toolkit.geocompose.MapView
+import com.arcgismaps.toolkit.geocompose.MapViewpointOperation
 import com.esri.arcgismaps.sample.manageoperationallayers.components.MapViewModel
 import com.esri.arcgismaps.sample.sampleslib.components.SampleTopAppBar
 
 /**
  * Main screen layout for the sample app
  */
 @Composable
-fun MainScreen(sampleName: String, application: Application) {
+fun MainScreen(sampleName: String) {
     // create a ViewModel to handle MapView interactions
-    val mapViewModel = MapViewModel(application)
+    val mapViewModel = MapViewModel(LocalContext.current.applicationContext as Application)
 
     Scaffold(
         topBar = { SampleTopAppBar(title = sampleName) },
         content = {
             Column(
                 modifier = Modifier.fillMaxSize().padding(it)
             ) {
-                // composable function that wraps the MapView
-                ComposeMapView(
+                MapView(
                     modifier = Modifier.fillMaxSize().weight(1f),
-                    mapViewModel = mapViewModel,
+                    arcGISMap = mapViewModel.arcGISMap,
+                    viewpointOperation = MapViewpointOperation.Set(viewpoint = mapViewModel.viewpoint)
```

Comment:
```suggestion
                    viewpointOperation = MapViewpointOperation.Set(viewpoint = Viewpoint(39.8, -98.6, 5e7))
```

I think we can directly set the Viewpoint here instead of creating a variable for it.